playground/mocks/data/idp/idx/error-authenticator-enroll-passcode-change-rsa.json

```diff
@@ -0,0 +1,184 @@
+{
+  "stateHandle": "02PynGLTh-s-KsG_QASp54JGH3ux3U1dDZRR6KWEQa",
+  "version": "1.0.0",
+  "expiresAt": "2021-01-07T01:54:21.000Z",
+  "intent": "LOGIN",
+  "messages": {
+    "type": "array",
+    "value": [
+      {
+        "message": "Wait for token to change, then enter the new tokencode.",
+        "i18n": {
+          "key": "idx.error.on-prem.token.change",
```

Comment:
Has this error code been added yet?

Reply:
I thought this was going to be added as part of your change? above is mock response

Reply:
Just wanted to make sure we didn't collide on this. Will be added in my next misc PR after the behemoth is merged :)